src/animation/gravity.ts

```diff
@@ -56,21 +39,33 @@ const applyForceForStep = (
  * animation at any time. The animation will stop when the mover has reached the
  * attractor as well.
  */
-const DEFAULT_THRESHOLD = { min: 10, max: 1000 };
 
-export const gravity = (params: GravityParams): [() => void] => {
+interface Gravity1DParams {
+  config: {
+    moverMass: number;
+    attractorMass: number;
+    r: number;
+    initialVelocity?: number;
```

Comment:
Now that we're modeling things along a line, we can give a user more intuitive inputs. They don't need to know anything about the vector math – just give us the pair of masses, the distance between masses, and an initial velocity if you feel like it. This seems simpler to me and will probably be more familiar for most users seeking a `from` / `to` style animation.